gapii/cc/spy.cpp

```diff
@@ -98,11 +96,14 @@ const int32_t kSuspendIndefinitely = -1;
 thread_local gapii::CallObserver* gContext = nullptr;
 
 }  // anonymous namespace
-
+#include <windows.h>
```

Comment:
Shouldn't this be inside a 
#if TARGET_OS == GAPID_OS_WINDOWS?

Reply:
Thanks! I actually had moved it up, but forgot to remove this one. Fixed now.